harness/determined/cli/checkpoint.py

```diff
@@ -2,7 +2,8 @@
 from argparse import Namespace
 from typing import Any, List, Optional
 
-from determined import cli
+from determined import cli, errors
+from determined.cli.errors import CliError
```

Comment:
new code should follow the [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports).

In this particular case, the conflict with `errors`, you'll need to bend the rules, and we don't have a clear standard for those cases, but I happen to like `cli.errors.CliError` best in this case.